data/operators.csv

```diff
@@ -364,3 +364,4 @@ NETSTYLE A. LTD,cloud,,unsafe,43945,59342
 Galaxy Broadband,ISP,started,unsafe,139879,59342
 Chilean Government Network (Red de Conectividad del Estado),ISP,signed + filtering,safe,17147,67265
 Equinix Metal,Cloud,signed + filtering peers,partially safe,54825,774
+NetSyst,ISP,signed + filtering,safe,209097,100
```

Comment:
your AS rank is 10343 per https://asrank.caida.org/asns?asn=209097&type=search

replace "100" with this value